### DISCUSSION TITLE
Setup authentification system

### DISCUSSION THREAD

**Discussion by KikyoC** [plan-player-analytics/Plan#3666] [Q&A]
Hello, I just installed plan with reverse proxy, but I cannont enable authentification system and everyone can access to informations. How can I setup that ? (/plan register --code says that an error occured but nothing in the console)

**Comment by AuroraLS3** [comment]
After enabling https on the reverse proxy you can set Plan to use that to enable login https://github.com/plan-player-analytics/Plan/wiki/SSL-Certificate-%28HTTPS%29-Set-Up#if-behind-a-proxy
